packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -176,7 +179,7 @@ async function fetchLatestSnapshotCore(
                 errorType: "access denied",
             }, tokenFetchOptions.previousError);
         }
-        const storageToken = await storageTokenFetcher(tokenFetchOptions, "TreesLatest");
+        const storageToken = await storageTokenFetcher(tokenFetchOptions, "TreesLatest", true);
```

Comment:
Do you guys do the convention of adding a comment for the param name? I.e.,:
`true /* alwaysRecordTokenFetchTelemetry */`

Reply:
No

Reply:
I do and prefer it :)